bin/reth/src/main.rs

```diff
@@ -21,27 +28,63 @@ fn main() {
         std::env::set_var("RUST_BACKTRACE", "1");
     }
 
-    if let Err(err) = Cli::parse_args().run(|builder, _| async {
-        let handle = builder.launch_node(EthereumNode::default()).await?;
+    let queue = Arc::new(Mutex::new(TransactionsPriorityQueue::new(1000)));
+    let queue_clone = Arc::clone(&queue);
+
+    if let Err(err) = Cli::parse_args().run(move |builder, _| async {
+        let handle = builder
+            .node(EthereumNode::default())
+            .extend_rpc_modules(move |ctx| {
+                let forwarder_required = ctx.config().bitfinity_import_arg.tx_queue;
+                if forwarder_required {
+                    // Add custom forwarder with transactions priority queue.
+                    queue
+                        .blocking_lock()
+                        .set_size_limit(ctx.config().bitfinity_import_arg.tx_queue_size as _);
+
+                    let forwarder = BitfinityTransactionsForwarder::new(queue);
+                    ctx.registry.set_eth_raw_transaction_forwarder(Arc::new(forwarder));
```

Comment:
How does Reth use this forwarder? Should it be used automatically?

Reply:
The forwarder mechanism is extension point, provided by reth to allow custom transaction manipulations. By default the forwarder is None, so we should set it.